aiohttp_admin/admin.py

```diff
@@ -80,3 +83,62 @@ def setup_admin_handlers(admin, admin_handler, static_folder, admin_conf_path):
     add_route('DELETE', '/logout', a.logout, name='admin.logout')
     add_static('/static', path=static_folder, name='admin.static')
     add_static('/config', path=admin_conf_path, name='admin.config')
+
+
+class AdminOnRestHandler:
+
+    template = 'admin_on_rest.jinja2'
+
+    def __init__(self, admin, *, resources, loop, schema):
+        self._admin = admin
+        self._loop = loop
+        self.schema = schema
+
+        for r in resources:
+            r.setup(self._admin, URL('/'))
+        self._resources = tuple(resources)
+
+    @property
+    def resources(self):
+        return self._resources
+
+    async def index_page(self, request):
```

Comment:
Lets add at least one test case for this handle

Reply:
\+